db/migrate/20210601181158_unique_partnerships.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class UniquePartnerships < ActiveRecord::Migration[6.1]
+  def change
+    add_index :partnerships, %i[school_id lead_provider_id cohort_id], unique: true, name: "unique_partnerships"
+  end
```

Comment:
I think this would fail if we had data, right? We don't yet in production, worth checking staging

Reply:
Yup, I was not very keen on writing consolidation migration hoping we have no data. I've checked both production and staging, but few days ago. I'll double check...